features/dataset-view.feature

```diff
@@ -213,43 +213,43 @@ Feature: a user visit the dataset page
 		And I should see an "select.selectPageSize" element
 		And I should see "Columns:"
 		And I should see "File Description"
-		And the "description" checkbox is unchecked
+		And the "description" checkbox is checked
 		And I should see "Sample ID"
-		And the "sample_id" checkbox is checked
+		And the "sample_id" checkbox is unchecked
 		And I should see "Data Type"
 		And the "type_id" checkbox is checked
 		And I should see "File Format"
-		And the "format_id" checkbox is checked
+		And the "format_id" checkbox is unchecked
 		And I should see "Size"
 		And the "size" checkbox is checked
 		And I should see "Release Date"
-		And the "date_stamp" checkbox is checked
+		And the "date_stamp" checkbox is unchecked
 		And I should see "Download Link"
 		And the "location" checkbox is checked
 		And I should see "File Attributes"
-		And the "attribute" checkbox is unchecked
+		And the "attribute" checkbox is checked
 		And I should see a button "Save changes" with no link
 		And I should see a button "Close" with no link
 
-	@ok @pageSize @files @javascript
+	@wip @pageSize @files @javascript @pr464
```

Comment:
I think the `@wip` tag should be removed here. When we use `@wip` in our future work with Behat tests then this test will also run which we won't want it to.